app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,296 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+/* //= require_tree .
+//= require_self
+//= require application */
+
+body {
+  background-color: #e5e5e5;
+}
+
+.users {
+  background-color: #fff;
+  padding: 10px;
+  margin: 10px;
+  border-radius: 5px;
+}
+
+.users h2 {
+  font-size: 30px;
+  font-weight: bold;
+  margin: 0;
+  padding: 0;
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.usersContent {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+
+.each_user {
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+  padding: 10px;
+  border-radius: 5px;
+  gap: 15px;
+  border: 1px solid rebeccapurple;
+}
+
+.each_user:hover {
+  background-color: #f5f5f5;
+  cursor: pointer;
+}
+
+.each_user img {
+  width: 50px;
+  height: 50px;
+  border: 1px solid #758ce2;
+  border-radius: 50%;
+}
+
+.each_user .name {
+  display: flex;
+
+  /* flex-direction: column; */
```

Comment:
- [ ] Kindly remove commented code from your project as it makes code hard to read.